website/docs/secrets/setup-sops.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Setup Sops
+hide_title: true
+---
+
+
+# Managing secrets with Sops
```

Comment:
You dont explain anywhere what this page is going to help the user do or achieve. It's important to set context. The user needs to know that this is the guide to follow if they want to enable sops on an existing cluster.